crates/oracle/src/lib.rs

```diff
@@ -138,6 +145,10 @@ pub mod pallet {
     #[pallet::getter(fn authorized_oracles)]
     pub type AuthorizedOracles<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, Vec<u8>, ValueQuery>;
 
+    /// If an oracle operator has feed a value in this block
+    #[pallet::storage]
+    pub type HasDispatched<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, bool, ValueQuery>;
```

Comment:
This will cause our current oracle client to fail since it's making multiple calls rather than combining everything into a single vec. So please mark the PR & commit as breaking. Also, it might be a good idea to update the client prior to merging this so we can't get blocked on it

Reply:
I'm also fine to revert the last commit FWIW or we could make this so we allow one call per unique key?